### PR TITLE
Try to use a Shipwright build as custom task

### PR DESCRIPTION
**Pull request by vdemeester**
Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by vdemeester** [comment]
Needs https://github.com/openshift-pipelines/pipelines-as-code/pull/656 to be in 😛

File: .tekton/build-nodejs-ex.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: shipwright.io/v1alpha1
+kind: Build
```
Comment (adambkaplan) [inline .tekton/build-nodejs-ex.yaml:2]: Does this imply that PaC needs RBAC to create Build objects?

Comment (vdemeester) [reply .tekton/build-nodejs-ex.yaml:2]: This is not used so far as even with https://github.com/openshift-pipelines/pipelines-as-code/pull/656, pac won't apply this type. *But* yes, if PaC was apply it, we would need the rbac to do it. *And* if we use embedded spec (if supported), I think that would be similar.

**Comment by vdemeester** [comment]
```
    build-with-shipwright-fd822-shipwright:
      pipelineTaskName: shipwright
      status:
        completionTime: '2022-05-11T15:53:14Z'
        conditions:
          - lastTransitionTime: '2022-05-11T15:53:14Z'
            message: >-
              the Build is not registered correctly, build: nodejs-ex,
              registered status: False, reason: SpecOutputSecretRefNotFound
            reason: BuildRegistrationFailed
            status: 'False'
            type: Succeeded
        extraFields:
          buildRunName: build-with-shipwright-fd822-shipwright-hmbfn
        startTime: '2022-05-11T15:53:14Z'
```
We are "getting" there 😝

**Comment by adambkaplan** [comment]
looks like Shipwright is expecting a push secret for the Build on the cluster.

**Pull request closed without merging**